devops/windows/base-docker-wsl/conf/env/conf-env-high-1-pc.template

```diff
@@ -0,0 +1,116 @@
+export SPEC_SETTINGS=high
+export NODE1=${NODE1}
+export NODE1_IP=${NODE1_IP}
+
+# export NODE2=${NODE2}
+# export NODE2_IP=${NODE2_IP}
+
+# export NODE3=${NODE3}
+# export NODE3_IP=${NODE3_IP}
+
+export STACK_NAME=jempi
+
+export PROJECT_DEVOPS_DIR=${PROJECT_DIR}
+export PROJECT_DATA_DIR=${PROJECT_DIR}/docker_data/data
+export PROJECT_DATA_APPS_DIR=${PROJECT_DIR}/docker_data/data-apps
+export PROJECT_DATA_MONITOR_DIR=${PROJECT_DIR}/docker_data/data-monitor
+
+export REGISTRY_NODE_IP=${REGISTRY_NODE_IP}
+export IMAGE_REGISTRY=${IMAGE_REGISTRY}
+
+export DATA_REGISTRY_DIR=${PROJECT_DIR}/data-registry
+export DATA_HAPROXY_DIR=${PROJECT_DIR}/data-haproxy
+export DATA_NGINX_DIR=${PROJECT_DATA_DIR}/data-nginx
+export DATA_KEYCLOAK_TEST_SERVER_DIR=${PROJECT_DATA_DIR}/data-keycloak-test-server
+export DATA_KAFKA_01_DIR=${PROJECT_DATA_DIR}/kafka-01
+export DATA_KAFKA_02_DIR=${PROJECT_DATA_DIR}/kafka-02
+export DATA_KAFKA_03_DIR=${PROJECT_DATA_DIR}/kafka-03
+export DATA_DGRAPH_ZERO_01_DIR=${PROJECT_DATA_DIR}/dgraph/zero-01
+export DATA_DGRAPH_ALPHA_01_DIR=${PROJECT_DATA_DIR}/dgraph/alpha-01
+export DATA_DGRAPH_ALPHA_02_DIR=${PROJECT_DATA_DIR}/dgraph/alpha-02
+export DATA_DGRAPH_ALPHA_03_DIR=${PROJECT_DATA_DIR}/dgraph/alpha-03
+export DATA_POSTGRESQL_DIR=${PROJECT_DATA_DIR}/postgres
+
+
+export DATA_DIR_ASYNC_RECEIVER=${PROJECT_DATA_APPS_DIR}/async_receiver
```

Comment:
Do we need java app specific items here since we will be running this on the host?

Reply:
the docker stack running in wsl must only have config for the "external" services - kafka, draph etc

All Java app stuff must be removed ----- this was copy paste thing that needs to be cleaned up